src/components/queue/QueueList/QueueList.tsx

```diff
@@ -4,15 +4,18 @@ import {
     CircularProgress,
     Grid,
     Stack,
+    Tooltip,
+    Button,
     Typography,
 } from "@mui/material";
-import Button from "@components/shared/Button";
 import CreateTicketDialog from "@components/queue/CreateTicketDialog";
 import QueueListItem from "@components/queue/QueueListItem";
 import {useTickets} from "@util/queue/hooks";
-import {Queue, Ticket} from "@util/queue/api";
+import {Queue, Ticket, TicketStatus} from "@util/queue/api";
```

Comment:
Added the ticket status import because it is required for the backend deletion API call check